components/Steps/__snapshots__/ReviewWarningNoteConfirmation.spec.tsx.snap

```diff
@@ -3,13 +3,13 @@
 exports[`Review Warning Note Confirmation it should rennder the correct text if the decision was no 1`] = `
 <DocumentFragment>
   <div
-    class="govuk-panel govuk-panel--confirmation lbh-panel govuk-!-margin-bottom-9"
+    class="govuk-panel govuk-panel--confirmation lbh-panel"
   >
-    <h1
-      class="govuk-panel__confirmation"
+    <div
+      class="govuk-panel__body"
     >
       This Warning Note has been closed
```

Comment:
probably irrelevant to this pr, but this should all be sentence case: "Warning Note" -> "warning note"

Reply:
Yeah I've seen a few instances of this – I will raise at our refinement